src/config.ts

```diff
@@ -401,12 +409,12 @@ export const depletion: IDataTypeConfig = {
   ]
 };
 
-export const prismDrug: IDataTypeConfig = {
+export const drugScreen: IDataTypeConfig = {
   id: 'drug',
-  name: 'PRISM Drug Screen',
+  name: 'Drug Screen',
   tableName: 'drugscore',
   query: 'drug_score',
-  screenTypes: ['Prism', 'GDSC', 'CCLE'],
+  screenTypes: [{id: 'Prism', text: 'Prism (Prism drug repurposing screen)'}, {id: 'GDSC', text: 'GDSC (Genomics of Drug Sensitivity in Cancer)'}, {id: 'CCLE', text: 'CCLE (Cancer Cell Line Encyclopedia)'}],
```

Comment:
That does not work, since there might be different number of screens in the database.